### ISSUE TITLE
Open Pose caffe model not returning shape of mat?

### ISSUE THREAD

**Issue by kwatters** [bytedeco/javacv#1090]
Hi,  I've been working on creating an example that uses the COCO model that is trained for body pose detection.  
I've been working off the OpenCV example: https://docs.opencv.org/3.4.3/d7/d4f/samples_2dnn_2openpose_8cpp-example.html

I have the model files loading as expected (I think)

```
    net = readNetFromCaffe(protoTxt, model);
    Scalar scalar = new Scalar(0, 0, 0, 0);
    Mat inputBlob = blobFromImage(srcMat, 1.0, new Size(w, h), scalar, false, false);
    log.info("Input blob! {} ", inputBlob);
    net.setInput(inputBlob);
    Mat result = net.forward();
     int nparts = result.size(1);
```

At this point,  I'm expecting "nparts" to be 19.. however it's coming up as 3x that as 57... 

Has anyone tried using OpenPose with JavaCV ?  I suspect there's something funky about the shape of the returned Mat from the net.forward() ..  

I'm trying this with opencv-3.4.2-1.4.2  ... maybe I should update to the latest?

**Comment by saudet** [comment]
What does toString() return on that?

I don't think a minor version is going to make any difference, no.

**Comment by saudet** [comment]
Then again, this example is for OpenCV 3.4.3, and isn't available for 3.4.2, so I'm guessing it doesn't work for 3.4.2 anyway. Let me know if you get the same thing with 3.4.3 though! Thanks
